pkg/eden/eden.go

```diff
@@ -353,6 +353,56 @@ func StatusEVEQemu(pidFile string) (status string, err error) {
 	return utils.StatusCommandWithPid(pidFile)
 }
 
+const (
+	prlctlPath = "prlctl"
+)
+
+//Run parallels cli tool and catch all errors
+func PrlctlOutput(args ...string) (string, error) {
+	if runtime.GOOS != "darwin" {
+		return "", fmt.Errorf("parallels works only on \"darwin\" platform")
+	}
+
+	var stdout, stderr bytes.Buffer
+
+	cmd := exec.Command(prlctlPath, args...)
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err := cmd.Run()
+
+	stderrString := strings.TrimSpace(stderr.String())
+
+	if _, ok := err.(*exec.ExitError); ok {
+		err = fmt.Errorf("calling prlctl: %s", stderrString)
+	}
+
+	return stdout.String(), err
+}
+
+//Parallels run with arguments
+func Prlctl(args ...string) error {
+	_, err := PrlctlOutput(args...)
+	return err
+}
+
+//StartEVEParallels function run EVE in parallels
+func StartEVEParallels(vmName string, parallelsCpus int, parallelsMem int) (err error) {
+	Prlctl("create", vmName, "--distribution", "ubuntu", "--no-hdd")
```

Comment:
Should we create VM on start action? We have setup step for this I think. Can you please elaborate.